From 44c9f185997a20cb493070260cf7206e47585f0e Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 28 Apr 2015 23:39:29 +0100
Subject: [PATCH 10/10] Disable WIP: Xcode generator

.. due to linking errors since it was turned
into a plugin:

undefined reference to 'qbs::XcodeNativeGenerator::XcodeNativeGenerator()'
undefined reference to 'qbs::XcodeSimpleGenerator::XcodeSimpleGenerator()'

I've not got the time to investigate and fix this
and Jake probably will soon enough anyway.
---
 src/lib/corelib/tools/projectgeneratormanager.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/lib/corelib/tools/projectgeneratormanager.cpp b/src/lib/corelib/tools/projectgeneratormanager.cpp
index f5d0f81..4c62fc0 100644
--- a/src/lib/corelib/tools/projectgeneratormanager.cpp
+++ b/src/lib/corelib/tools/projectgeneratormanager.cpp
@@ -39,9 +39,6 @@
 #include <QDirIterator>
 #include <QLibrary>
 
-#include <generators/xcode/xcodenativegenerator.h>
-#include <generators/xcode/xcodesimplegenerator.h>
-
 namespace qbs {
 
 using namespace Internal;
@@ -63,8 +60,6 @@ ProjectGeneratorManager *ProjectGeneratorManager::instance()
 ProjectGeneratorManager::ProjectGeneratorManager()
 {
     QList<QSharedPointer<ProjectGenerator> > generators;
-    generators << QSharedPointer<XcodeNativeGenerator>::create();
-    generators << QSharedPointer<XcodeSimpleGenerator>::create();
     foreach (QSharedPointer<ProjectGenerator> generator, generators) {
         m_generators[generator->generatorName()] = generator;
     }
-- 
2.3.5

